assignment.js

```diff
@@ -65,6 +68,19 @@ Use the following test cases to confirm your program meets the success criteria
 console.log('Problem 2:')
 
 // Add your code below this line
+const numbers = [57, 13, 4, 38]
+let currentLargestNumber = 0
+
+for (i=0; i < numbers.length; i++){
```

Comment:
@KatieGarrison15  nice approach and great job getting this working. Also good use of using your knowledge of arrays to use `i` as an index to implement the logic of keeping track of the largest number. 

Take a look at an alternative [approach used here](https://github.com/jsd20200127/hw-01-js-basics-control-flow-loops-solution/blob/master/assignment.js#L98) that tests against a single condition to determine the current largest number.